app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,83 @@
+class VendorsController < ApplicationController
+  def index
+    uri = URI(rmapi_path)
+
+    # Transform query params to what the EBSCO RM-API expects
+    uri.query = URI.encode_www_form(
+      search: params[:q],
+      orderby: params[:orderby] || params[:q] ? 'relevance' : 'vendorname',
+      count: params[:count] || 25,
+      offset: params[:offset] || 1
+    )
+
+    # Make the RM-API request and parse the response body
+    response = make_request(uri)
+    data = JSON.parse(response.body)
+
+    # Return the list of vendors
+    if response.code === '200'
```

Comment:
do we wan to check for `200` specifically here, or would the more idiomatic `response.ok?` be sufficient?

Reply:
`response.ok?` gives me an undefined method error.

Reply:
Went with `response.message === 'OK'`